theme/tutorial.less

```
@@ -221,6 +221,24 @@ body#docs.tutorial {
     width: 100%;
 }
 
+.ui.modal{
+    .ams-embed {
+        width: 100%;
+        height: 30rem;
+        max-width: 80%;
+        max-height: 50vh;
+    }
+}
+
+.ui.modal {
+    .yt-embed {
+        width: 100%;
+        height: 30rem;
+        max-width: 80%;
+        max-height: 50vh;
+    }
+}
```

```suggestion
.ui.modal {
    .ams-embed, .yt-embed {
        width: 100%;
        height: 30rem;
        max-width: 80%;
        max-height: 50vh;
    }
}
```

since you wan the same css to apply to both it just needs a comma separate list like this~ This also helps since you most likely want the size to be consistent between the two so when we need to make a change in the future we don't forget to update one

---

good point! thanks for suggestion, should have remembered that!